docker.Makefile

```diff
@@ -4,7 +4,7 @@
 # Makefile for developing using Docker
 #
 
-+.PHONY: build_docker_image build clean cross dev
++.PHONY: build_docker_image build_linter_image build clean test cross dev lint
```

Comment:
I think some of these are not strictly needed here, because I think depending on a PHONY target always forces it to run, but not a problem.